src/components/Documentation/Card/index.tsx

```diff
@@ -11,15 +11,16 @@ interface Props {
   links: LinkItem[]
   title: string
   Icon: React.ComponentType<{ className?: string }>
+  anchorId?: string
 }
 
-const DocumentationCard: React.FC<Props> = ({ links, title, Icon }) => {
+const DocumentationCard: React.FC<Props> = ({ links, title, Icon, anchorId }) => {
   if (!title || !Icon || !links || typeof links !== "object") {
     return null
   }
 
   return (
-    <div className="w-full lg:w-1/3 p-3 group">
+    <Link href={`/docs#${anchorId}`} id={anchorId} className="w-full lg:w-1/3 p-3 group scroll-mt-25">
```

Comment:
Can you just compute the final href value to avoid **undefined** in the url if there is no anchorId?

`href="/docs#undefined"`


```suggestion
    <Link href={`/docs${anchorId ? ("#"+anchorId) : ""}`} id={anchorId} className="w-full lg:w-1/3 p-3 group scroll-mt-25">
```

Reply:
```href={`/docs#${anchorId ? anchorId : ""}`}```

Is this good?